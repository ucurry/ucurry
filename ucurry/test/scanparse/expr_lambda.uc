-- def -> expr -> lambda
\(int -> int) x -> x + 1;
