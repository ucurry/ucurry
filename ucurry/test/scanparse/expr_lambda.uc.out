Printing Roundtrip
\((int -> int))x -> x + 1;
